### ISSUE TITLE
could you release the trained model

### ISSUE THREAD

**Issue by amiltonwong** [DylanWusee/pointconv_pytorch#4]
Hi, @DylanWusee ,

I've tried this package and the training was converged at epoch 12 (accuracy = 0.888710) and then evaluated it, the resulting total accuracy is 0.894246, which is lower than the reported one (0.925)

Would you release the trained model for better comparison?

THX!

**Comment by anyuecq25** [comment]
hi,I have the same problem.The accuracy at epoch 101 is 0.913306. This is the best result that I can get.  Please share the pre-trained model.

**Comment by JinyangGuo** [comment]
Hi,

I also tried to train the model and only get 91.6% for the ModelNet40 classification task. Could you please release the pre-trained model?

**Comment by DylanWusee** [comment]
Hi,

Thank you for your interest in our work. The original implementation is in Tensorflow. It is possible that the pytorch version can give slightly different results. I will try to release a pre-trained model ASAP.

Thank you very much.
Best,
Dylan

**Comment by JinyangGuo** [comment]
Hi Dylan,

Thank you for your response. I also notice that the input point cloud of this code are 2048 points instead of 1024 points. I print the shape of the points variable in both training code and the testing code. Do I misunderstand the code?

Regards,
Jinyang

**Comment by GYE19970220** [comment]
> 嗨迪伦，
> 
> 感谢您的答复。我还注意到该代码的输入点云为2048点而不是1024点。我在训练代码和测试代码中都打印了点变量的形状。我会误解代码吗？
> 
> 此致
> 金阳

Excuse me, How to modify the number of input points?

**Comment by yechaowang** [comment]
I got 91.6% for the ModelNet40.......

**Comment by DylanWusee** [comment]
The pretrained model is relased.

The previous code has a bug in the inverse density scale, which might be the reason that the model does not reach the accuracy in the paper.

However, I have updated the code. And, the new code should be able to achieve 92+%.

Beside, I also upload a pretrained model. You can use it as a reference.
